core/services/ocr3/plugins/ccip/commit/plugin.go

```diff
@@ -146,6 +146,8 @@ func (p *Plugin) Observation(ctx context.Context, outctx ocr3types.OutcomeContex
 		"tokenPrices", len(tokenPrices),
 		"observerInfo", p.cfg.ObserverInfo)
 
+	// tmp
```

Comment:
nit: make it harder to ignore
```suggestion
	// TODO: replace temporary code
```

Reply:
nice, i forgot that